### DISCUSSION TITLE
Suppress split of datacube into continents

### DISCUSSION THREAD

**Discussion by frankweiser** [davidfrantz/force#118] [Q&A] [answered]
Hi all,

I am currently working on a project on the Canary Islands. When processing Landsat data to Level 2, the data is split into two subfolders, "Europe" and "Africa". Even partially overlapping tiles are sorted into only one of the two. This makes the next steps (time series analysis) much harder since both have their own datacube_definition.prj and the time series is less dense due to the split.

Is there any way to suppress this split into continents and keep all data in one folder?

I am using FORCE 3.7.0 on Mint 19.1, projection is GLANCE7

So far, I didn't find a solution, any help is appreciated.

Best,
Frank

**Comment by davidfrantz** [comment]
Hi @frankweiser,

can you please post a screenshot of this sorting issue? I do not fully understand what the layout of this issue is.

Regarding the 2 datacubes: you can select one of the sub-grids with ``PROJECTION = GLANCE7-EU`` or ``GLANCE7-AF``.
As your area of interest seems to extend between two zones, however, GLANCE7 is probably less suitable for you. You can always define your own custom projection in WKT format.

Either specifically use the WKT representation of one of the GLANCE7 projections (see ``datacube-definition.prj``). Or use an entirely different projection.

This here is an example of the official EU-wide equal area projection:

``PROJECTION = PROJCS["ETRS89 / LAEA Europe",GEOGCS["ETRS89",DATUM["European_Terrestrial_Reference_System_1989",SPHEROID["GRS 1980",6378137,298.257222101,AUTHORITY["EPSG","7019"]],AUTHORITY["EPSG","6258"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4258"]],PROJECTION["Lambert_Azimuthal_Equal_Area"],PARAMETER["latitude_of_center",52],PARAMETER["longitude_of_center",10],PARAMETER["false_easting",4321000],PARAMETER["false_northing",3210000],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AUTHORITY["EPSG","3035"]]``

You could center this projection to your study area by modifying the ``latitude_of_center`` and ``longitude_of_center`` values, probably setting ``false_easting`` and ``false_northing`` to 0.

Cheers,
David

**Reply by frankweiser** [reply]
Dear David, 
thanks for the quick reply! Attached are some screenshots. The folder structure is basically the problem: During Level 2 processing, the level 2 images are sorted into subfolders called "Africa" and "Europe" which contain their own datacube_definition and have very different tile X/Ys 
![folder_sctructure](https://user-images.githubusercontent.com/88429417/128165436-a63a2405-4147-415d-bd9f-9252da7647cd.png)
![folder_structure2](https://user-images.githubusercontent.com/88429417/128165481-3c278c10-01e0-4a5f-8b10-cd545942791a.png)
![folder_structure3](https://user-images.githubusercontent.com/88429417/128165483-c079b32b-fbd1-4f59-b533-5e8c3b45d98b.png)

In reality, the images overlap, so images from both Africa and Europe should be in the same folder for easier handling. Here is a quick map illustrating this for Gran Canaria, the true colour scene is from the Africa folder, the overlapping scene with the blue tint is from Europe. 
![FORCE_GC2](https://user-images.githubusercontent.com/88429417/128166018-c462d0f9-5fa6-45fe-a700-d1e8a3d41ecb.png)

Thanks for the tip, setting my own projection sounds good, I'll try it tomorrow.

Best,
Frank

**Comment by davidfrantz** [comment, marked as answer]
Hmm, 
most likely, some of the images intersect with the area for which the grid was designed, and other images do not, 
I believe in these edge cases, it is really best to work with a different projection.
Cheers,
David

**Reply by frankweiser** [reply]
Dear David, 
I tried the projection you suggested, seems like it is working fine now. Processing will still take a while, but the images completed so far are simply in the level 2 folder, without a continent folder in between. 

Thank you for the quick help!

Best,
Frank
